content/docs/command-reference/repro.md

```diff
@@ -255,14 +256,40 @@ You can now check that `dvc.lock` and `count.txt` have been updated with the new
 information: updated dependency/output file hash values, and a new result,
 respectively.
 
+## Example: Specific stage
+
+> This example continues the previous one.
+
+We can use `dvc repro` to reproduce a pipeline partially till a specific stage.
+Let's add a line to `text.txt` , for example:
+
+```
+...
+4321
+```
+
+Now, specify the `count` stage as a `target` with `dvc repro`. This results in
```

Comment:
Let's use filter here, otherwise this is not really a partial reproduction as the whole pipeline will be reproduced. Also because it's the same as the next example, basically.

Reply:
Or introduce an intermediate step if you are trying to show that using a target can also reproduce the previous stages, so that the next example is not so repetitive (in it's effects).

Reply:
Actually it would be ideal to have that intermediate step so it can be used with --downstream in the next example, to show the contrast between this example (target the middle stage which runs the previous ones) vs downstream (BTW also targets a stage, but the behavior is different because of the --downstream option).

Reply:
BUT we shuold probably extract that part to a separate issue to keep this PR simple.

Reply:
> downstream (BTW also targets a stage

This makes me think that another solution is to combine the 2 examples by basically just clarifying in the old one that a stage name target is being used (combined with --downstream). For example the title could be **Example: Downstream from a target stage**.

Reply:
WDYT?

Reply:
Combining them both is better I think. As both example requires a stage as target. This doesn't require much changes in current example. Maybe we can add a note(or separate paragraph) that says without `--downstream` pipeline would be reproduces until the target and not further.

Reply:
OK. Ideally examples should focus on a single thing but I guess in this case we can try the approach.